frameworks/PHP/mako/mako.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
+RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php > /dev/null && \
+    apt-get update -yqq > /dev/null && apt-get upgrade -yqq > /dev/null
+
+RUN apt-get install -yqq nginx git unzip \
```

Comment:
I tested to have faster builds. But it have problems.
Please udpate to:
`RUN apt-get update -yqq > /dev/null && apt-get install -yqq nginx git unzip \`

Reply:
the `apt-get update` line is on line 6, are you wanting me to remove that or keep both?